examples/arithmetics/example/example.calc

```diff
@@ -1,14 +1,18 @@
-module example1
+MODULE example1
```

Comment:
Please don't forget to revert the example before merging.

Reply:
I'd propose to leave it to demonstrate the ability to make keywords case-insensitive.